mysql-test/suite/galera/include/galera_load_provider.inc

```diff
@@ -1,8 +1,3 @@
---echo Loading wsrep provider ...
-
---disable_query_log
```

Comment:
this line was removed unnecessarily.

Reply:
Good catch and this causes port 13002 to appear, I'll fix this.

Reply:
Fixed.